config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,26 @@
+if Rails.env.test? || Rails.env.cucumber?
```

Comment:
Cucumber? I think you're using rspec, so you don't need last part of this condition. Just pay attention to what you copy-paste

Reply:
Yep, you are right 😬